### PR TITLE
edit.js has checkboxes for grabbable pseudo-properties

### PR DESCRIPTION
**Pull request by sethalves**
- edit.js has checkboxes for grabbable pseudo-properties

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: cca4a93b2b604b036588e32135d8062f251e5057 available at: https://gist.github.com/68f8335fa0fb9aedafa3<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7340.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7340.dmg

**Comment by sethalves** [comment]
to test:
- use the scripts from this PR
- create boxes and models, adjust the grab-related properties
- try grabbing things and make sure the psuedo-properties work correctly

File: examples/html/entityProperties.html [hunk 1]
```diff
@@ -243,6 +243,47 @@
 
         }
 
+        function isGrabbableByDefault(properties) {
+            return properties.type == "Box" ||
+                properties.type == "Sphere" ||
+                properties.dynamic
+        }
```
Comment (AndrewMeadows) [inline examples/html/entityProperties.html]: I'm curious: why are boxes and spheres "grabbable by default"?

Comment (sethalves) [inline examples/html/entityProperties.html]: apparently they no longer are.  I'll fix it

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1f8a55da3abbaec197ab407dc949e396596b1405 available at: https://gist.github.com/68f8335fa0fb9aedafa3<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7340.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7340.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: d811c4f6b53ff7abf18fc7539b658aaff0947067 available at: https://gist.github.com/68f8335fa0fb9aedafa3<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7340.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7340.dmg

**Comment by cozza13** [comment]
I tested and it works

**Pull request merged**